src/main/java/ch/interlis/iox_j/validator/Validator.java

```diff
@@ -389,7 +387,7 @@ public void validate(ch.interlis.iox.IoxEvent event) {
 				throw e;
 			}
 		} else if (event instanceof ch.interlis.iox.EndBasketEvent){
-		    clearCurrentBid();
+		    cleanupCurrentBasket();
```

Comment:
Q: Könnte es Sinn machen auf https://github.com/claeis/iox-ili/pull/119 aufzubauen, da wir dort auch einen Mechanismus für den Basket-Change einführen?

Reply:
Die generischen Wertebereiche werden im first pass validiert, in https://github.com/claeis/iox-ili/pull/119 passiert die Validierung jedoch im second pass. Daher gibt es keine Abhängigkeiten zwischen diesen Änderungen.